keras/layers/convolutional.py

```diff
@@ -1447,9 +1464,16 @@ class ZeroPadding2D(Layer):
     '''Zero-padding layer for 2D input (e.g. picture).
 
     # Arguments
-        padding: tuple of int (length 2)
+        padding: tuple of int (length 2) or tuple of int (length 4) or dictionary
+            For symmetric padding tuple of int (length 2)
             How many zeros to add at the beginning and end of
-            the 2 padding dimensions (axis 3 and 4).
+            the 2 padding dimensions (rows and cols).
+            For asymmetric padding tuple of int (length 4)
+            How many zeros to add at the beginning and at the end of
+            the 2 padding dimensions (rows and cols).
+            (top_pad, bottom_pad, left_pad, right_pad) or
+            {'top_pad': top_pad, 'bottom_pad': bottom_pad, 'left_pad': left_pad, 'right_pad': right_pad}
+            If any key is missing, default value of 1 will be used for the missing key.
```

Comment:
Change to 0. Also please format code elements in docstring with "`" (tuples. dicts).